tests/NuGet.Services.AzureSearch.Tests/SearchService/SearchTextBuilderFacts.cs

```diff
@@ -102,6 +102,46 @@ public void ThrowsWhenTermIsTooBig(string query)
             }
         }
 
+        public class Autocomplete : FactsBase
```

Comment:
I did not see tests for AzureSearchQueryBuilder. In case that there are not, IMO i think it would be good to add a set of tests for that area.

Reply:
Good question. `AzureSearchQueryBuilder` should only be used by `SearchTextBuilder`, it is effectively only an implementation detail. All the unit tests for `AzureSearchQueryBuilder` are covered by `SearchTextBuilder`'s unit tests.

I'll make this clearer by making `AzureSearchQueryBuilder` a nested type on `SearchTextBuilder`.

Reply:
`AzureSearchQueryBuilder` is now nested inside of `SearchTextBuilder`.